packages/pn-commons/src/components/TitleBox.tsx

```diff
@@ -37,16 +37,34 @@ export default function TitleBox({
   children,
 }: Props) {
   return (
-    <Grid container mt={mtGrid} sx={sx}>
-      {title && <Grid item xs={12} mb={mbTitle}>
-        <Typography variant={variantTitle}>{title}</Typography>
-      </Grid>}
-      {subTitle && <Grid item xs={12} mb={mbSubTitle}>
-        <Typography variant={variantSubTitle} sx={{ fontSize: '18px' }} component={typeof subTitle !== "string" ? "div" : "p"}>
-          {subTitle}
-        </Typography>
-      </Grid>}
-      <Grid item xs={12}>
+    <Grid id="container" aria-orientation="horizontal" tabIndex={0} container mt={mtGrid} sx={sx}>
+      {title && (
+        <Grid id="item" item xs={12} mb={mbTitle}>
+          <Typography
+            component={'h1'}
+            key={'recapiti-title'}
+            id={'title'}
+            role="heading"
+            aria-label={title.toString()}
```

Comment:
Not a good idea using `toString()` method. It gives "`[object Object]`" if you use react nodes instead of string in `title` parameter. is better to use a separate optional prop such ariaLabel: string instead.